### PR TITLE
Validate parameter values

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

**Comment by volodymyrss** [comment]
this addresses https://github.com/oda-hub/dispatcher-plugin-integral/issues/33 and is a step towards https://github.com/oda-hub/dispatcher-app/issues/100

needed now since it keep breaking requests

File: cdci_data_analysis/analysis/instrument.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def check_instrument_query_role(self, query_obj, product_type, roles, par_dic):
     def get_html_draw(self, prod_name, image,image_header,catalog=None,**kwargs):
         return self.get_query_by_name(prod_name).get_html_draw( image,image_header,catalog=catalog,**kwargs)
 
-    def get_par_by_name(self,par_name):
+    def get_par_by_name(self,par_name, validate=False):
```
Comment (volodymyrss) [inline cdci_data_analysis/analysis/instrument.py:276]: not sure if this is needed really

File: cdci_data_analysis/analysis/instrument.py [hunk 2]
```diff
@@ -366,6 +370,12 @@ def set_pars_from_form(self,par_dic,logger=None,verbose=False,sentry_client=None
             self.set_pars_from_dic(par_dic,verbose=verbose)
             #DONE
             q.set_done(debug_message=str(debug_message))
+        except RequestNotUnderstood as e:
```
Comment (volodymyrss) [inline cdci_data_analysis/analysis/instrument.py:373]: we could add an empty instrument test here maybe.

File: cdci_data_analysis/analysis/instrument.py (hunk 1 above)
Comment (burnout87) [reply cdci_data_analysis/analysis/instrument.py:277]: Currently is not doing anything so it is actually not needed, but we might want to implement it in the future

Comment (burnout87) [reply cdci_data_analysis/analysis/instrument.py:277]: So it's good to have it

**Review by burnout87 — CHANGES_REQUESTED** [review]

**Comment by volodymyrss** [comment]
I must have accidentially merged it, sorry.

**Pull request merged**